server/src/main/java/org/opensearch/index/remote/RemoteStoreUtils.java

```diff
@@ -116,4 +136,43 @@ static String longToUrlBase64(long value) {
         String base64Str = Base64.getUrlEncoder().encodeToString(hashBytes);
         return base64Str.substring(0, base64Str.length() - 1);
     }
+
+    static long urlBase64ToLong(String base64Str) {
+        byte[] hashBytes = Base64.getUrlDecoder().decode(base64Str);
+        return ByteBuffer.wrap(hashBytes).getLong();
+    }
+
+    /**
+     * Converts an input hash which occupies 64 bits of memory into a composite encoded string. The string will have 2 parts -
+     * 1. Base 64 string and 2. Binary String. We will use the first 6 bits for creating the base 64 string.
+     * For the second part, we will use the next 14 bits. For eg - A010001010100010.
+     */
+    static String longToCompositeUrlBase64AndBinaryEncodingUsing20Bits(long value) {
+        return longToCompositeBase64AndBinaryEncoding(value, 20);
```

Comment:
Instead of this specific function with 20 bits, why don't we call `longToCompositeBase64AndBinaryEncoding` wherever required?

Reply:
Not the renaming of this method, the original comment was to understand need of this method. The `hash` method of FNV_1A_COMPOSITE_1 can directly call `longToCompositeBase64AndBinaryEncoding(value, 20)`, right?

Reply:
This method is being called at multiple places in test class, hence kept the method with name having "using20bits". Let me use it directly.